runtime/plugin/tohtml.lua

```diff
@@ -0,0 +1,22 @@
+if vim.g.loaded_2html_plugin ~= nil then
+  return
+end
+vim.g.loaded_2html_plugin = true
+
+vim.api.nvim_create_user_command('TOhtml', function(params)
+  local winnr = vim.api.nvim_get_current_win()
+  ---@type string
+  local args = params.args
+  local html = require('tohtml').win_to_html(winnr)
+  if args == '-' then
+    for _, line in ipairs(html) do
+      vim.api.nvim_out_write(line .. '\n')
+    end
+  elseif args == '' then
+    local outfile = vim.fn.tempname() .. '.html'
+    vim.fn.writefile(html, outfile)
+    vim.cmd.split(outfile)
+  else
+    vim.fn.writefile(html, args)
+  end
```

Comment:
can't all this be moved to the `tohtml.lua` module? all this file should do (if anything at all) is define a convenience user command for `lua vim.tohtml(curwin, {write = false})` (or somesuch).